lib/theme-options/live-options.php

```diff
@@ -0,0 +1,30 @@
+<?php
+// SITE OPTIONS
+$prefix = '_igv_';
+$suffix = '_options';
+
+$page_key = $prefix . 'live' . $suffix;
+$page_title = 'Live Page Options';
+$metabox = array(
```

Comment:
The expectation here is that the pages are semi automatically driven. They don't act as Pages with any content just urls for templates. So to override different ones you need some options. I think grouping those options is more helpful for the admin user as they get to the edit one click from the dashboard rather than 2 clicks and a scroll and into a page editor that doesn't really make any sense. There are 2 other pages like this page